apps/nuxt3/app.vue

```diff
@@ -1,55 +1,55 @@
 <template>
-    <div>
-        <NuxtWelcome />
-        <vue-particles
-            id="tsparticles"
-            :options="options"
-            :particles-init="particlesInit"
-        />
-    </div>
+  <div>
+    <NuxtWelcome />
+    <vue-particles
+      id="tsparticles"
+      :options="options"
+      :particles-init="particlesInit"
+      @particles-loaded="particlesLoaded"
+    />
+  </div>
 </template>
-<script>
+<script setup lang="ts">
 import { loadFull } from "tsparticles";
+import type { ISourceOptions, Container } from "tsparticles-engine";
 
-export default {
-    name: "NuxtTutorial",
-    data() {
-        return {
-            options: {
-                background: {
-                    color: "#fff"
-                },
-                fullScreen: {
-                    enable: true,
-                    zIndex: -1
-                },
-                particles: {
-                    color: {
-                        value: "#000"
-                    },
-                    links: {
-                        color: "#000",
-                        enable: true
-                    },
-                    move: {
-                        enable: true
-                    },
-                    number: {
-                        value: 100
-                    }
-                }
-            }
-        };
+const options = {
```

Comment:
Instead of using `as ISourceOptions`, it's better to define the type in the declaration `const options: ISourceOptions`